promptulate/schema.py

```diff
@@ -28,6 +28,91 @@ def type(self) -> str:
         """Type of the message, used for serialization."""
 
 
+class BaseStreamIterator:
+    """
+    This class is an iterator for the response stream from the LLM model.
+
+    Attributes:
+        response_stream: The stream of responses from the LLM model.
+        return_raw_response: A boolean indicating whether to return the raw response
+        or not.
+    """
+
+    def __init__(
+        self,
+        response_stream,
+        parse_content: callable([[Any], [str, str]]),
```

Comment:
You should add relevant docs to show how to pass parse_content callback.

Reply:
ok